phoenix-core/src/it/java/org/apache/phoenix/calcite/CalciteIT.java

```diff
@@ -1315,6 +1315,17 @@ public void initTable() throws Exception {
                         {6L, "00A323122312312"},
                         {8L, "00A423122312312"}})
                 .close();
+
```

Comment:
Calcite seems to function a little bit differently with respect to CURRENT VALUE, so I thought it would be wise to include a case for it. Plus it's the only case in CalciteIT covering CURRENT VALUE.
See: https://issues.apache.org/jira/browse/PHOENIX-3437